backend/pokemons/helpers.py

```diff
@@ -28,17 +28,17 @@ def save_pokemon(poke_id):
     return pokemon
 
 
-def get_pokemons_points_sum(poke_ids):
+def get_pokemons_points_sum(names):
     points_sum = 0
-    for poke_id in poke_ids:
-        pokemon = Pokemon.objects.filter(poke_id=poke_id).first()
+    for name in names:
+        pokemon = Pokemon.objects.filter(name=name).first()
```

Comment:
From the performance perspective this is not a good practice. You will be making a new db query for each pokemon. In a real world project you have to assume someone can pass a lot of items to this function therefore causing it to be very costly. A better way to do this is by fetching all the pockemons in a single query and them iterating in this list.

Reply:
Be aware that by doing this you will need to do some changes to ensure the case where the pokemon is not in the db still works